.github/workflows/api-test.yml

```diff
@@ -88,6 +88,8 @@ jobs:
         case:
           - name: Tenant
             class: org.apache.dolphinscheduler.api.test.cases.TenantAPITest
+          - name: Project
+            class: org.apache.dolphinscheduler.api.test.cases.ProjectAPITest
```

Comment:
I find out you adding three new cases for api-test, but you just add one of them, please add the others to here to make it work

Reply:
Thanks for the reminder, it has been fixed